src/styles/_header.scss

```
@@ -37,5 +37,12 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+
+    transition: color 0.3s ease,
+      transform $transition-ease-duration;
```

```suggestion
      transform $transition-ease-duration;
```
You haven't transform in the header, only color. Delete this line.
Ease is a default value, you can delete it

---

oops, Ctrl + C/Ctrl + V failed me :(